test/tests/functional/server_tests.js

```diff
@@ -127,6 +127,42 @@ exports['Should correctly connect server to single instance and execute insert']
   }
 }
 
+exports['Should correctly connect server to single instance and execute insert (now with compression)'] = {
+  metadata: { requires: { topology: ["single", "snappyCompression"] } },
```

Comment:
looks like this environment was accidentally depended upon here, but not defined in the runner